tests/test_encode_decode.py

```diff
@@ -1,12 +1,86 @@
+import base64
 import quopri
 import unittest
 
 from hypothesis import given, strategies as st
 
 
+# function which adds padding, to make len(payload) a multiple of 4
+def add_padding(payload):
+    if len(payload) % 4 != 0:
+        padding = b"\0" * ((-len(payload)) % 4)
+        payload = payload + padding
+    return payload
+
+
 class TestBase64(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/base64.html
-    pass
+    # @given(
+    #     payload=st.binary(),
+    #     altchars=(st.none() | st.binary(min_size=2, max_size=2)),
+    #     validate=st.booleans(),
+    # )
```

Comment:
Need to uncomment, and use the hypothesis strategy defined on top instead of the one below.

Reply:
If you want to delete the commented-out code, I'd be happy to merge this using the strategies below :smile:

Reply:
Hey @Zac-HD! I have removed the commented out code. 😄